content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -334,3 +344,7 @@ The `distribute-cpus-across-cores` option can be enabled by adding
 `distribute-cpus-across-cores=true` to the `CPUManager` policy options.
 It cannot be used with `full-pcpus-only` or `distribute-cpus-across-numa` policy
 options together at this moment.
+
+The `prefer-align-cpus-by-uncorecache` option can be enabled by adding the
+`prefer-align-cpus-by-uncorecache` to the `CPUManager` policy options. It cannot
+be used with `distribute-cpus-across-numa` or `distribute-cpus-across-cores`
```

Comment:
suggestion:
```
if incompatible options are used, the kubelet will fail to start with the error explained in the logs.
```

Reply:
Thanks @ffromani 
Applied the suggested changes, please take a look when you get the chance, thanks!